h/api/search/index.py

```diff
@@ -147,7 +147,7 @@ def _prepare(self, annotation):
         # FIXME: this should not use the same presenter as we use to render
         # annotations for clients. It is useful in the mean time until we get rid of
         # the legacy ElasticSearch annotation storage.
-        data = presenters.AnnotationJSONPresenter(self.request, annotation).asdict()
+        data = presenters.AnnotationSearchIndexPresenter(self.request, annotation).asdict()
         data['target'][0]['scope'] = [annotation.target_uri_normalized]
```

Comment:
Ditto above. The reason this is separate is because we didn't want to add this to `AnnotationJSONPresenter`. There's no reason not to add it to `AnnotationSearchIndexPresenter`.